.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
+  - repo: https://github.com/adamchainz/django-upgrade
+    rev: 1.29.1
+    hooks:
+      - id: django-upgrade
+        args: [--target-version=4.2]
```

Comment:
The tool looks for the Django version in `pyproject.toml`. We don't have one, so defaults to 2.2. So let's specify it here, and remember to update when also bumping the Django version in `base-requirements.txt`.

https://django-upgrade.readthedocs.io/en/latest/options.html#cmdoption-target-version

Reply:
> remember to update when also bumping 

😬